### DISCUSSION TITLE
Priorities for our dear developer and his team

### DISCUSSION THREAD

**Discussion by helgews** [TabularEditor/TabularEditor3#272] [Other Topics]
@bernatagulloesbrina : I really like how you combine the power of TE with all the features the tabular model has to offer. You are pushing it to the "limit", but with clear purpose. On top of it all, you are sharing your work in repos - that is something I hope to achieve some time in the future (when I have some original ideas of my own 😄). I hope you succeed with your designs, and would love to help if I see some errors or have suggestions.

At the same time, its come to level where it benefits only a few (?) tabular developers. And sometimes its not clear at first wether its a bug/issue. I see Daniel has brought out a new tag - script assistance (good). 

I guess what I'm trying to say, is that I **really** look forward to some of the features posted on the roadmap, #12 . And I wonder if @otykier & team can multi-task and do everything at once (like it seems based on his level of activity).

These are just my thoughts at the moment - I hope I didn't offend anyone!

**Comment by bernatagulloesbrina** [comment]
Completely agree, if there's a better place to get "script assistance" I'll
gladly post it there!

**Comment by otykier** [comment]
You are raising a valid point, @helgews . In general, all questions are welcome on the GitHub site. Broadly speaking, our order of priority is the following:

1. Dedicated support (Enterprise customers)
2. Bugs (these have different levels of priority internally, based on how impactful they are)
3. Roadmap items
4. Feature requests/enhancements
5. General questions

Often times, however, a question can be answered in just a few minutes without requiring any investigation. I would not want to postpone answering such questions just because I have other stuff with higher priority to work on. @bernatagulloesbrina’s post from earlier is a good example: I believed I could quickly answer the first question, and so I did, however the follow-up question was much more complicated, linking to a 140 line script etc. I’m hoping that Bernat is not expecting an answer to that as quickly as the first one :-)

We are in the process of ramping up both our support and development staff, so in any case, we expect to be able to deliver the features listed in the roadmap within the planned quarters.

**Comment by bernatagulloesbrina** [comment]
Thank you for clarifying Daniel!

Also not all questions must be answered by Daniel. It has already happened
that other users have joined the exchange.

Hopefully one day I'll have enough command to answer some questions myself

Regards,
